picard/util/tags.py

```diff
@@ -40,186 +41,802 @@
+        is_file_info=True,
+        is_hidden=True,
+        is_preserved=True,
+        is_tag=False,
+    ),
+    TagVar(
+        'bits_per_sample',
+        shortdesc=N_('Bits per sample'),
+        is_file_info=True,
+        is_hidden=True,
+        is_preserved=True,
+        is_tag=False,
+    ),
+    TagVar(
+        'catalognumber',
+        shortdesc=N_('Catalog Number'),
+    ),
+    TagVar(
+        'channels',
+        shortdesc=N_('Channels'),
+        is_file_info=True,
+        is_hidden=True,
+        is_preserved=True,
+        is_tag=False,
+    ),
+
+    TagVar(
+        'comment',
+        shortdesc=N_('Comment'),
+    ),
+    TagVar(
+        'compilation',
+        shortdesc=N_('Compilation (iTunes)'),
+    ),
+    TagVar(
+        'composer',
+        shortdesc=N_('Composer'),
+    ),
+    TagVar(
+        'composersort',
+        shortdesc=N_('Composer Sort Order'),
+    ),
+    TagVar(
+        'conductor',
+        shortdesc=N_('Conductor'),
+    ),
+    TagVar(
+        'copyright',
+        shortdesc=N_('Copyright'),
+    ),
+    TagVar(
+        'datatrack',
+        shortdesc=N_('FIXME:datatrack'),
+        is_hidden=True,
+        is_tag=False,
+    ),
+    TagVar(
+        'date',
+        shortdesc=N_('Date'),
+    ),
+    TagVar(
+        'director',
+        shortdesc=N_('Director'),
+    ),
+    TagVar(
+        'dirname',
+        shortdesc=N_('Directory name'),
+        is_hidden=True,
+        is_preserved=True,
+        is_tag=False,
+    ),
+    TagVar(
+        'discid',
+        shortdesc=N_('Disc Id'),
+    ),
+    TagVar(
+        'discnumber',
+        shortdesc=N_('Disc Number'),
+    ),
+    TagVar(
+        'discpregap',
+        shortdesc=N_('FIXME:discpregap'),
+        is_hidden=True,
+        is_tag=False,
+    ),
+    TagVar(
+        'discsubtitle',
+        shortdesc=N_('Disc Subtitle'),
+    ),
+    TagVar(
+        'djmixer',
+        shortdesc=N_('DJ-Mixer'),
+    ),
+    TagVar(
+        'encodedby',
+        shortdesc=N_('Encoded By'),
+    ),
+    TagVar(
+        'encodersettings',
+        shortdesc=N_('Encoder Settings'),
+    ),
+    TagVar(
+        'engineer',
+        shortdesc=N_('Engineer'),
+    ),
+    TagVar(
+        'extension',
+        shortdesc=N_('File extension'),
+        is_hidden=True,
+        is_preserved=True,
+        is_tag=False,
+    ),
+    TagVar(
+        'file_created_timestamp',
+        shortdesc=N_('File created timestamp'),
+        is_hidden=True,
+        is_preserved=True,
+        is_tag=False,
+    ),
+    TagVar(
+        'file_modified_timestamp',
+        shortdesc=N_('File modified timestamp'),
+        is_hidden=True,
+        is_preserved=True,
+        is_tag=False,
+    ),
+    TagVar(
+        'filename',
+        shortdesc=N_('File name'),
+        is_hidden=True,
+        is_preserved=True,
+        is_tag=False,
+    ),
+    TagVar(
+        'filepath',
+        shortdesc=N_('File Path'),
+        is_hidden=True,
```

Comment:
I think this could get a `is_tag=False`. Same with `filesize` and `length`. Currently they all show up in e.g. tag list editors. But selecting them does not make much sense. E.g. adding one of those to the preserved tags list.

Reply:
Done in https://github.com/metabrainz/picard/pull/2637/commits/d880e115383846c068b5f678f1a5ace9c82b8bb2